dashboard/app/models/circuit_playground_discount_application.rb

```diff
@@ -24,6 +24,25 @@ class CircuitPlaygroundDiscountApplication < ApplicationRecord
   belongs_to :user
   belongs_to :circuit_playground_discount_code
 
+  enum unit_6_intention: {
+    no: 1,
+    yes1718: 2,
+    yes1819: 3,
+    yesAfter: 4,
+    unsure: 5,
+  }
```

Comment:
These keys need to be the same as the values in our radio buttons. Initially decided not to try sharing this via shared_constants.rb, but not sure that's the right decision.

Reply:
I think part of why I avoided shared_constants.rb is it felt like overkill for something just used on this one page. I wonder if it would make sense to just share it via script data.

Reply:
Thinking more, even if I pass down the keys in some way, I still need to map them to string (i.e. `no` to `i18.eligibilityYearNo()`). It's not clear to me that any other approach is actually going to make things cleaner.

Reply:
Given the temporary nature of this page, I think the method used here is sufficient.